src/oncall_admin/api.py

```diff
@@ -89,6 +89,37 @@ def on_post(self, req, resp):
         cursor = connection.cursor(db.dict_cursor)
         cursor.execute('INSERT INTO `user` (`name`, `active`) VALUES (%s, TRUE)', username)
         user_id = cursor.lastrowid
+
+        ldap_info = ldap_user.get_ldap_user(username)
```

Comment:
Users often use oncall-admin to avoid setting up LDAP sync. To be more friendly to this use-case (avoiding needless "User not found in ldap" log messages), add a check to skip this code if the LDAP section is not configured.

Reply:
I've added a test to only try to fetch ldap informations when `ldap_user.connection` is set